lib/vtls/schannel.c

```diff
@@ -383,7 +402,7 @@ schannel_connect_step1(struct connectdata *conn, int sockindex)
   }
   memset(connssl->ctxt, 0, sizeof(struct curl_schannel_ctxt));
 
-  host_name = Curl_convert_UTF8_to_tchar(hostname);
+  host_name = Curl_convert_UTF8_to_tchar(conn->host.name);
```

Comment:
I think this change breaks HTTPS-proxy support. Why was this changed from hostname which is used in all other places regarding certificate verification in this function?